File: app/src/main/java/objectone/zstas/com/object1/MainActivity.java
package objectone.zstas.com.object1;

import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import objectone.zstas.com.object1.ui.main.MainFragment;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.main_activity);
        if (savedInstanceState == null) {
            getSupportFragmentManager().beginTransaction()
                .replace(R.id.container, MainFragment.newInstance())
                .commitNow();
        }
    }
}

File: presentation/src/main/java/com/zstas/objectone/MainActivity.kt
package com.zstas.objectone

import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)

        bar {
            println("WAZZZZAAAAAP")
        }
    }

    private fun bar(a: () -> Unit) {
        a.invoke()
    }
}